support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -190,11 +190,6 @@ export function Checkout({ geoId, appConfig }: Props) {
 		}
 	}
 
-	/**
```

Comment:
I'm removing this comment as hopefully this change makes the cookie a more accurate indicator. I'm not sure what the effort entailed to "send it down from the server" would be